pkg/kube/cluster-init.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/ash
+#
+# Copyright (c) 2023 Zededa, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+K3S_VERSION=v1.26.3+k3s1
+KUBEVIRT_VERSION=v0.59.0
+LONGHORN_VERSION=v1.4.2
+CDI_VERSION=v1.56.0
+
+INSTALL_LOG=/var/log/install.log
```

Comment:
Is there an upper bound of how much it will write here? This is overlayfs hence memory AFAIK

Reply:
Even while it is running fine? That becomes how much in a year?

Reply:
Approx 32MB but you are right not point wasting that memory. I will move to /var/lib